augly/tests/video_tests/base_unit_test.py

```diff
@@ -118,7 +118,6 @@ def evaluate_class(
                 output_path=tmpfile.name, seed=seed, metadata=metadata, **kwargs
             )
             self.assertTrue(os.path.exists(tmpfile.name))
```

Comment:
why did you. remove this line?
```suggestion
            self.assertTrue(os.path.exists(tmpfile.name))

```